custom_components/spotcast/__init__.py

```diff
@@ -41,7 +41,7 @@
     WS_TYPE_SPOTCAST_PLAYER,
     WS_TYPE_SPOTCAST_PLAYLISTS,
 )
-from .helpers import async_wrap, get_cast_devices, get_spotify_devices
+from .helpers import async_wrap, get_cast_devices, get_spotify_devices, get_spotify_media_player
```

Comment:
please see merge conflicts, due to the line getting long, I moved to a parentheses block import for the helper methods